bibedit/record_conference_paper.xml

```diff
@@ -23,6 +23,10 @@ along with INSPIRE; if not, write to the Free Software Foundation, Inc.,
     <subfield code="a">VOLATILE:e.g 10.1103/example.doi.for.inspire</subfield>
     <subfield code="2">VOLATILE:DOI</subfield>
   </datafield>
+  <datafield tag="035" ind1=" " ind2=" ">
+    <subfield code="a">VOLATILE:external record identifier</subfield>
+    <subfield code="9">VOLATILE:source</subfield>
```

Comment:
beside TeX-keys which are not really external, at the moment we have in 035__9: 
arXiv 
CERNKEY 
CDS 
DESY 
KEKSCAN 
OSTI 

-- 
Florian Schwennsen 
Deutsches Elektronen-Synchrotron DESY 
Building 01 Room O1.446 
phone: +49-40-8998-6190 

> From: "Samuele Kaplun" <notifications@github.com>
> To: "inspirehep/inspire" <inspire@noreply.github.com>
> Cc: "Florian Schwennsen" <florian.schwennsen@desy.de>
> Sent: Monday, 8 June, 2015 13:33:38
> Subject: Re: [inspire] bibedit: update record templates (#88)

> In bibedit/record_conference_paper.xml :
>> @@ -23,6 +23,10 @@ along with INSPIRE; if not, write to the Free Software
> > Foundation, Inc.,
> >      <subfield code="a">VOLATILE:e.g 10.1103/example.doi.for.inspire</subfield>
> >      <subfield code="2">VOLATILE:DOI</subfield>
> >    </datafield>
> > +  <datafield tag="035" ind1=" " ind2=" ">
> > +    <subfield code="a">VOLATILE:external record identifier</subfield>
> > +    <subfield code="9">VOLATILE:source</subfield>

> What are the known sources so far? Are they enumerable? (thinking about when we
> move this to JSONSchema)

> —
> Reply to this email directly or view it on GitHub .